conductr_cli/sandbox_run_jvm.py

```diff
@@ -467,6 +468,11 @@ def download_sandbox_image(image_dir, package_name, artefact_type, image_version
     try:
         bintray_auth = bintray_resolver.load_bintray_credentials(raise_error=False)
 
+        if sandbox_version.version_parts(image_version) >= (2, 1, 0):
```

Comment:
@markusjura - I see. Thanks for explanation. I've used `is_conductr_on_private_bintray` as you've suggested.